bundles/org.openhab.binding.qbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="qbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>Qbus Bridge</label>
+		<description>This bridge represents a Qbus client</description>
+		<config-description>
+			<parameter name="addr" type="text" required="true">
+				<label>Hostname</label>
+				<description>IP address or hostname of Qbus server, usually 'localhost'</description>
+				<default>localhost</default>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="sn" type="text" required="true">
+				<label>Serial Number</label>
+				<description>Serial number of the CTD controller</description>
+			</parameter>
+			<parameter name="port" type="integer" required="false">
+				<label>Bridge Port</label>
+				<description>Port to communicate with Qbus server, default 8447</description>
+				<default>8447</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="serverCheck" type="integer" required="false">
+				<label>Server Check</label>
+				<description>Time to check communication with Qbus Server (min), default 10. If set to 0 or left empty, no
+					refresh
+					will be scheduled</description>
```

Comment:
Can you fix the line break? Put all into one line and then run `mvn spotless:apply`.